source/documentation/infrastructure/getting-access.md

```diff
@@ -0,0 +1,113 @@
+# Accessing the Infrastructure
+
+## VPN
+
+All connections must be made via the GDS VPN. Please contact your local service desk for access.
+
+## Bastions
+
+The bastion servers act as a gateway to their respective clusters + databases.
+That is to say:
+
+- To access any Staging database or server, you must access via the staging bastion.
+- To access any Production database or server, you must access via the production bastion.
+
+### Accessing the Bastion
+
+#### Secrets
+
+[Extract the SSH Key secrets][getting-a-secret] for the bastion server:
+
+- staging secret name: `keys/govwifi-staging-bastion-key`
+- production secret name: `keys/govwifi-bastion-key`
+
+Find out the Elastic IPs for the bastion servers. You can do this by going into the AWS console,
```

Comment:
Can you add somewhere the AWS account number(s) or link to where these can be found, so one can configure the assume role on AWS console or `aws vault` (but you don't need to explain how to configure this!)?